spec/schemas.spec.js

```diff
@@ -143,8 +144,9 @@ describe('schemas', () => {
     config = Config.get('test');
   });
 
-  afterEach(() => {
-    config.database.schemaCache.clear();
+  afterEach(async () => {
+    await config.database.schemaCache.clear();
+    await TestUtils.destroyAllDataPermanently(false);
```

Comment:
`destroyAllDataPermanently` should already be being called by helper.js:

https://github.com/parse-community/parse-server/blob/d13bdbcea6d7103842fe79e82810bb20d11e3de3/spec/helper.js#L194-L238

currently, we call it in beforeEach and if we're on postgress we ALSO call it in afterEach.

In my open PR, I change it so that it is alway run in afterEach.

I'd think we'd want to change it so it is run in beforeAll and in afterEach,

in berforeAll to make sure that we're starting with a blank db in case the previous run left data due to a bad exit, or whatever.

And then in afterEach to make sure that each test cleans up data when it's done.

I don't see why we'd want more than that...

Reply:
`destroyAllDataPermanently(fast)` The fast parameter is probably used to make the tests run faster by deleting all objects instead of dropping the table (which drops indexes too).

I set it to false for the test spec.

Reply:
ok. let's see how it does!